facebook_connector.py

```diff
@@ -0,0 +1,90 @@
+from fbchat_archive_parser.parser import MessageHtmlParser
+import fnmatch
+from lxml import html
+import os
+import pandas as pd
+import re
+from urllib2 import urlopen
+
+_messages_file = None
+_fb_id_pattern = re.compile("(\\d+)@facebook\\.com")
+_html_tags_pattern = re.compile('<.*?>')
+_mapped_fb_ids = {}
+
+
+def initialize(dump_location='.'):
+    global _messages_file
+    fb_message_filename = 'messages.htm'
+    _messages_file = os.path.join(dump_location, fb_message_filename)
+    if not os.path.isfile(_messages_file):
+        print("""
+Please place the messages.htm file at {}
+or specify the location of the file in the function call
+""".format(os.path.abspath(_messages_file)))
+        _messages_file = None
+
+
+def resolve_user_id(user_id):
```

Comment:
Before I even review this method in depth, what're your thoughts on just returning it as a url without loading it, so https://facebook.com/<123> and in a follow up PR loading it?  That way we can get this PR in earlier without needing to review this more complex logic